com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -751,8 +752,6 @@ private void OnNetworkPreUpdate()
                     m_CurrentNetworkTimeOffset += Mathf.Clamp(m_NetworkTimeOffset - m_CurrentNetworkTimeOffset, -maxDelta, maxDelta);
                 }
             }
-
-            NotifyProfilerListeners();
```

Comment:
coming from: #641 — fix: fixed network activity occurring outside the core update getting missed in the profiler